CHECK_POWER_COMMANDS/Backstop_File_Class.py

```diff
@@ -266,4 +266,4 @@ def insert_errors( self, lr_file, violations_list):
             print('Copying ACIS-LoadReview.txt.ERRORS to ACIS-LoadReview.txt')
             mv_status = subprocess.run(['mv', 'ACIS-LoadReview.txt.ERRORS', 'ACIS-LoadReview.txt'])
         except:
-            print('The move command failed. Examine the ofls directory and look for the .ERRORS file.')
+            print('The move command failed. Examine the ofls directory and look for the .ERRORS file.', mv_status)
```

Comment:
Did you test this? If `subprocess` raises an exception then `mv_status` will not be defined and the `print` statement itself will fail. You might consider using a code linter to find code errors like this.